config-templates/linux-local-dev-segue-config.properties

```diff
@@ -73,8 +73,8 @@ FASTTRACK_GAMEBOARD_WHITELIST=ft_core_2017,ft_core_stage2
 
 # Security
 HMAC_SALT=fbf4c8996fb92427ae41e4649SUPER-SECRET-KEY896354df48w7q5s231a
-SESSION_EXPIRY_SECONDS_DEFAULT=432000
-SESSION_EXPIRY_SECONDS_REMEMBERED=1209600
+SESSION_EXPIRY_SECONDS_DEFAULT=1800
+SESSION_EXPIRY_SECONDS_REMEMBERED=1800
```

Comment:
Do we still need this?

Reply:
Okay, had a look into this. Doesn't look like I can change the DTOs without breaking things, we'd need to do something coordinated with the front-end I suspect...
Aside from the DTOs though, I have a set of shelved changes to strip out the rememberMe flag and associated expiry time property if we want to go down that route?
Alternatively, I could probably modify the cookie to store that flag and allow the refresh to support it. Not sure what alternative expiry time we'd want to assign for that though...